compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -247,6 +247,12 @@ impl ToStableHashKey<StableHashingContext<'_>> for MonoItem<'_> {
     }
 }
 
+#[derive(Debug, HashStable, Copy, Clone)]
+pub struct CodegenUnits<'tcx> {
```

Comment:
Nitpick: Calling this `CodegenUnits` seems slightly weird, since it contains both `codegen_units` and associated global info that is *not* codegen units.

(For a struct that exists to be returned by a single query, my first thought would have been to name it something boring derived from the name of the query.)

Reply:
yea... I couldn't come up with anything so I used that. `MonoItemPartitions` maybe?